scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -755,7 +755,8 @@ def apply_filters(self, **kwargs):
                     instance.add_filter("Command line platform filter", Filters.CMD_LINE)
 
                 if ts.platform_allow and plat.name not in ts.platform_allow:
-                    instance.add_filter("Not in testsuite platform allow list", Filters.TESTSUITE)
+                    if not (platform_filter and force_platform):
```

Comment:
Could be this `if` combined together with previous one? Like for example:
```python
if ts.platform_allow and \
        plat.name not in ts.platform_allow and \
        not (platform_filter and force_platform):
```

Reply:
Sure, I can. I wanted to separate this check to point, that this is somethin extra ... and long `if` line always looks ugly and are not easy to understand, when analyzing the code after some time